logbook-api/src/main/java/org/zalando/logbook/Strategy.java

```diff
@@ -0,0 +1,30 @@
+package org.zalando.logbook;
+
+import org.apiguardian.api.API;
+
+import java.io.IOException;
+
+import static org.apiguardian.api.API.Status.STABLE;
+
+@API(status = STABLE)
+public interface Strategy {
```

Comment:
TODO documentation? Just saying as the `README` teases this :wink:

Reply:
:man_facepalming:

Reply:
Fixed